manual/guides/uml.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: manual
+title: Generating UML Diagrams
+subtitle: "Generating UML Diagrams"
+permalink: /manual/guides/uml
+---
+
+Universal Modeling Language (UML) diagrams are a common way to visually
+represent software components (such as classes and functions). When reviewing
+code with someone new to the software, UML diagrams can be a useful tool.
+
+Rather than writing UML diagrams by hand, we can generate them from existing
+code using existing software tools. T
+
+| Language | Tools |
```

Comment:
This table is a little squished, so I'll make separate pr to update the bootstrap css and add some padding.

Reply:
![image](https://github.com/user-attachments/assets/4bdf84bf-740d-4f5c-af98-cabcdc683cc0)